packages/button/src/Button.tsx

```diff
@@ -18,7 +18,16 @@ type ButtonProps = ButtonHTMLAttributes<HTMLButtonElement> & {
   isLoading?: boolean;
   loadingText?: string | JSX.Element;
   size?: 'tiny' | 'small' | 'medium' | 'big';
-  kind?: 'default' | 'primary' | 'destructive' | 'minimal' | 'link' | 'close';
+  kind?:
+    | 'default'
+    | 'primary'
+    | 'destructive'
+    | 'minimal'
+    | 'link'
+    | 'close'
```

Comment:
What is the `close` variant for?

Reply:
Existing buttons used for modals 

<img width="858" alt="Screenshot 2023-08-23 at 10 50 12 AM" src="https://github.com/launchdarkly/launchpad-ui/assets/36527396/00e321df-36c4-4ba1-831c-92dc0f8eb835">

Reply:
Oh, we're adding a variant just for close buttons? 🤔 The `default` variant wasn't it?

Reply:
The `close` variant already existed. I did not add that. Should we remove it? I agree its very similar to `default`

Reply:
**Default**
<img width="865" alt="Screenshot 2023-08-23 at 10 54 54 AM" src="https://github.com/launchdarkly/launchpad-ui/assets/36527396/f1990278-3995-454e-af03-8e55617dc568">

Reply:
Oh, sorry, you're right. You did add a story which is why I'm seeing it in chromatic now!

Hmm I wonder where we use that variant. if we don't use it anywhere, we should remove it.

Reply:
well that was easy…

we only use it in _one_ place in the product: 
<img width="1071" alt="CleanShot 2023-08-23 at 13 56 18@2x" src="https://github.com/launchdarkly/launchpad-ui/assets/232396/f41aa90e-e69e-435a-a200-cb0b02839ea6">

Reply:
Its only used in 2 locations of the app 
![Screenshot 2023-08-23 at 10 57 43 AM](https://github.com/launchdarkly/launchpad-ui/assets/36527396/fd3c315e-2db1-406d-ab23-021f78de7ec0)

Reply:
And we use it in alerts, banners and snackbars…

Let's ignore it here for now. I'll take it to Launchpad.